documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -118,10 +152,21 @@ Linux devices on Raspberry Pi OS:
 | secondary UART
 |===
 
-NOTE: `/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
+`/dev/serial0` and `/dev/serial1` are symbolic links which point to either `/dev/ttyS0` or `/dev/ttyAMA0`.
+
+On the Raspberry Pi 5, `/dev/serial0` is a symbolic link that points to `/dev/ttyAMA10`. Due to changes in Bookworm, `/dev/serial1` does not exist by default on the Raspberry Pi 5. You can enable `serial1` by setting the following values in `confix.txt`:
+
+[source,bash]
+----
+dtparam=krnbt=off
+----
+
+TIP: This option may not work on all models in the future.
```

Comment:
"This option _is almost certainly not what you really want,_ and may not work on all models in the future."